.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules
 coverage
-
+.DS_Store
```

Comment:
Please remove

Reply:
I'm happy to -but this is a standard cruft meta-data file automagically created in mac environments - without it - most contributors on macs may inadvertently slip up and commit .DS_Store files.

Reply:
Yes I agree these files are annoying, but I think this should be part of developers own git config rather than polluting projects with this, especially since there are different files based on each OS and IDE.

Reply:
works for me!